docs/source/project_documentation/user_guide/project_structure.md

```diff
@@ -9,6 +9,39 @@ src/service_one/service_one_misc_definitions.yaml
 src/service_two/service_two.yaml
 ...
 ```
+src
+├── system
+│   └── system.yaml
+├── external
+|   ├── external_actors.yaml
+│   └── external_interface_messages.yaml
+├── service_one
+|   ├── service_one.yaml
+│   └── service_one_misc.yaml
+└── service_two
+    └── service_two.yaml
```

Comment:
I like the `tree` view. Please replace the content in lines 4-9 with this since it's just a duplicate of the content shown differently which can be confusing. Of course, that would also mean we should remove it from lines 12-22 so as not to have this content duplicated.